mifosng-android/build.gradle

```diff
@@ -147,6 +146,7 @@ tasks.withType(JavaCompile) {
 
 dependencies {
     implementation 'androidx.multidex:multidex:2.0.0'
+    implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
```

Comment:
where are we using this library?

Reply:
The dependencies was used in [maven { url 'http://dl.bintray.com/amulyakhare/maven' }](https://github.com/openMF/android-client/pull/1866/files#diff-f60af60b2e4a2537c2402faacea0730fc3f194c4359324498f39088f4cea0692L31) which generates error on compilation.